src/lists.rs

```diff
@@ -26,6 +26,8 @@ pub enum ListTactic {
     Horizontal,
     // Try Horizontal layout, if that fails then vertical
     HorizontalVertical,
+    // HorizontalVertical with a soft limit.
```

Comment:
Maybe make explicit that this is a character limit. An item limit would also be feasible.